### DISCUSSION TITLE
Why CPU and GPU max OC went down from when it was on KazushiMe ?

### DISCUSSION THREAD

**Discussion by gmxw** [hanai3Bi/Switch-OC-Suite#31] [Q&A]
My Switch (Mariko) was running perfectly fine with the CPU, GPU and RAM max overclock when it was on KasushiMe's GitHub, and just want to know why these clocks for CPU and GPU aren't available since it is on hanai3Bi's GitHub (version related ?).

It has been a while that the KasushiMe's GitHub for Switch-OC-Suite isn't available, so idk exacly what was the frequency for CPU but I know it was higher (maybe 2397Mhz ?) but for GPU I know it was 1305Mhz.

Again, I just want to know why those frenquencies aren't available since then.

PS : I tried to build it myself fisrt so I could probably try to edit those clocks myself for my switch, but didn't manage to do it, it alaways gives me an error. I want to try out some frequencies with my switch to push it to its limits, and I don't care if it brakes because of it.

**Comment by hanai3Bi** [comment]
you can do 2397 cpu if you enable cpu uv. 2397 cpu is removed on normal table because it's not gonna work for everyone, and for  some people it messes up other freq and breaks functionality, reason why it's only available on uv table.

and official gpu max clock is 1267, but 1267 is not safe for switch, but nevertheless it's available. 1305 is remove because it's not official clock, and it's power draw is too much and too dangerous for switch

**Reply by hanai3Bi** [reply]
if you want 1305 gpu, you can always build yourself. it will not be availabe on public release

**Reply by gmxw** [reply]
I enabled mariko cpu uv in the online configuration but still max I can select is 2295Mhz.

I tried to build the atmosphere folder from source, I always get this error
customize.cpp
make: g++: No such file or directory
make: *** [Makefile:42: build/./customize.cpp.o] Error 127

everything else builds perfectly fine

**Reply by hanai3Bi** [reply]
try reinstall everything. 2397 shows up fine for me.
and for buillding there's guide in readme file refer to that

**Reply by gmxw** [reply]
I will try again tomorrow, and thanks for your quick responses

**Reply by gmxw** [reply]
For 1397 cpu I had to also enable gpu uv for it to show up.

For building, I downloaded the entire repo with git, and still same issue. patch.py worked with no problem but building still shows me the same error.

I have some questions, how do you check the power draw and what is considered safe for switch ? Would 1305 gpu be safer with uv ? I managed to get 1267 gpu with 750mv without losing any performance in a very cpu and gpu demanding game.
On the DvfsTable, I only undertood what the clocks were but what are the other values in this table ?

**Reply by hanai3Bi** [reply]
1. NO
2. they are dvfs voltage calculation coefficients

**Comment by hanai3Bi** [comment]
safe gpu limits: 998~1075 for normal table, 1075 for slt table, 1228 for hiopt table

**Reply by hanai3Bi** [reply]
reason for this is not power draw, but gpu pmic current limit(10A)
